packages/Charts/src/demos/demo.ts

```diff
@@ -0,0 +1,73 @@
+import * as DG from 'datagrok-api/dg';
+import * as grok from 'datagrok-api/grok';
+
+
+export class Demo {
+  static async chordViewerDemo() {
+    const df = await grok.data.getDemoTable('energy_uk.csv');
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('ChordViewer');
+  }
+
+  static async globeViewerDemo() {
+    const df = await grok.data.getDemoTable('geo/earthquakes.csv');
+    const tableView = grok.shell.addTableView(df);
+    // timeout is for the table view to fully load, activate detectors and then start the globe viewer
+    setTimeout(() => {
+      tableView.addViewer('GlobeViewer');
+    }, 300);
+  }
+
+  static async groupAnalysisViewerDemo() {
+    const df = await grok.data.getDemoTable('demog.csv');
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('GroupAnalysisViewer');
+  }
+
+  static async radarViewerDemo() {
+    const df = await grok.data.getDemoTable('demog.csv');
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('RadarViewer');
+  }
+
+  static async sankeyViewerDemo() {
+    const df = await grok.data.getDemoTable('energy_uk.csv');
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('SankeyViewer');
+  }
+
+  static async sunburstViewerDemo() {
+    const df = await grok.data.getDemoTable('demog.csv');
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('SunburstViewer');
+  }
+
+  static async surfacePlotDemo() {
+    const df = await grok.data.getDemoTable('demog.csv');
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('SurfacePlot');
+  }
+
+  static timelinesViewerDemo() {
+    const df = DG.DataFrame.fromCsv(
+      `USUBJID, AESTDY, AEENDY, SEX, AGE
+      s1, 10/02/2018, 10/09/2018, F, 48
+      s2, 10/04/2018, 10/07/2018, M, 51
+      s3, 10/02/2018, 10/05/2018, F, 39
+      s4, 10/07/2018, 10/08/2018, M, 43`);
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('TimelinesViewer');
```

Comment:
A small suggestion: you can use the docking API to place a viewer in a more suitable position, for example:
```js
const viewer = tableView.addViewer('TimelinesViewer');
tableView.dockManager.dock(viewer, DG.DOCK_TYPE.DOWN);
```
It's up to you to decide on the most suitable position, I used this particular case because the demo data is small and we have an empty space under the table anyway

Reply:
hm, will think about a suitable position for the viewers in the demos, it may play a role in user perception